FirebaseTestProject/Views/MainTabView.swift

```diff
@@ -14,10 +14,11 @@ struct MainTabView: View {
                 .tabItem {
                     Label("Posts", systemImage: "list.dash")
                 }
-            NewPostForm()
-                 .tabItem {
-                     Label("New Post", systemImage: "plus.circle")
-                 }
+            PostsList(viewStyle: .favorites)
```

Comment:
`PostsList(viewStyle: .favorites)` does not compile:
> Argument passed to call that takes no arguments
> Cannot infer contextual base in reference to member 'favorites'

Looks like you forgot to include the `PostsList(viewStyle:)` initializer and the `PostData.ViewStyle` enumeration.

Reply:
Odd... I'll correct that!

Reply:
Resolved by [commit 98a62a3563639f771ec93899fab361f63dc747a6](https://github.com/benstone1/InstaCademy/pull/13/commits/98a62a3563639f771ec93899fab361f63dc747a6)